packages/fuels/tests/types_contracts.rs

```diff
@@ -2094,6 +2094,13 @@ async fn test_contract_std_lib_string() -> Result<()> {
             .call()
             .await?;
     }
+    {
+        let _resp = contract_methods
+            .echoes_dynamic_string(String::from("Hello Fuel"))
+            .with_tx_policies(TxPolicies::default().with_script_gas_limit(2000))
+            .call()
+            .await?;
+    }
```

Comment:
There should be a comment explaining why this test is so important. It is only too easy to say "oh we test this elsewhere, this should be removed".